src/app/api/api_v1/endpoints/detections.py

```diff
@@ -60,7 +60,7 @@ async def create_detection(
         min_length=2,
         max_length=settings.MAX_BBOX_STR_LENGTH,
     ),
-    azimuth: float = Form(..., gt=0, lt=360, description="angle between north and direction in degrees"),
+    azimuth: float = Form(..., gt=-1, lt=360, description="angle between north and direction in degrees"),
```

Comment:
Actually this will allow ]-1,360[ which could be problematic. I think you're looking for:

```suggestion
    azimuth: float = Form(..., ge=0, lt=360, description="angle between north and direction in degrees"),
```